src/guess.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { calc } = require('./calc');
+
+const rl = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let tries = 1;
+
+const guess = (digits) => {
+  rl.question(`${tries} try: `, (userInput) => {
+    const bulls = calc(userInput, digits)[0];
```

Comment:
You are calling the 'calc' function twice, consider storing the result in a variable and use array destructuring to assign values to 'bulls' and 'cows'.